src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -230,6 +221,11 @@ export default {
 		},
 		closeLightbox() {
 			this.showLightbox = false;
+		},
+		verifySpotlightIndex() {
+			const spotlightCookie = this.cookieStore.get('lh_spotlight') || null;
+			if (spotlightCookie) this.spotlightIndex = spotlightData.length - 1 <= Number(spotlightCookie) ? 0 : Number(spotlightCookie) + 1; // eslint-disable-line max-len
```

Comment:
Storing `Number(spotlightCookie)` in a const with the body of the `if` statement would make this line a little more readable and wouldn't need the linting disable.